pkg/kapp/clusterapply/delete_change.go

```diff
@@ -122,3 +157,14 @@ func descMessage(res ctlres.Resource) []string {
 	}
 	return []string{}
 }
+
+func (c DeleteChange) isInoperableResource() bool {
+
+	res := c.change.ExistingResource()
+	for _, r := range inoperableResourceList {
+		if r.Name == res.Name() && r.Kind == res.GroupKind().Kind && r.Group == res.GroupKind().Group {
```

Comment:
> 
```suggestion
		if r.Name == res.Name() && r.Kind == res.Kind() && r.Group == res.APIGroup() {
```
Looks like we have separate methods for Kind and Group so better to use them.